test/pylib/scylla_cluster.py

```diff
@@ -115,7 +115,7 @@ def make_scylla_conf(workdir: pathlib.Path, host_addr: str, seed_addrs: List[str
     '-m', '1G',
     '--collectd', '0',
     '--overprovisioned',
-    '--max-networking-io-control-blocks', '100',
+    '--max-networking-io-control-blocks', '1000',
```

Comment:
Why does a 2-shard node need more than 100 simultaneous connections?

Not a merge blocker, but we should understand why.

Reply:
The test uses nodes with --smp 3, --smp 4, --smp5.  In addition, since this is a configuration with different shard numbers, shard on the node with fewer shards will have connections from more than one pairing shards.

Reply:
It's 100 per shard. So something is wrong.